include/octo-logger-cpp/manager.hpp

```diff
@@ -39,12 +40,12 @@ class Manager
     ManagerConfigPtr config_;
     Log::LogLevel default_log_level_;
     std::shared_ptr<Logger> global_logger_;
+    std::shared_ptr<ContextInfo const> global_context_info_;
```

Comment:
Any reason for `std::shared_ptr` over `std::unique_ptr`?
Using `std::shared_ptr` implies sharing, this class is a singleton.

Reply:
added comment